cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/PreviewRunnerModule.java

```diff
@@ -185,11 +168,6 @@ protected void configure() {
     expose(AccessEnforcer.class);
     bind(ContextAccessEnforcer.class).toInstance(contextAccessEnforcer);
     expose(ContextAccessEnforcer.class);
-    bind(AccessControllerInstantiator.class).toInstance(accessControllerInstantiator);
-    expose(AccessControllerInstantiator.class);
-    bind(PermissionManager.class).toInstance(permissionManager);
-    expose(PermissionManager.class);
-    bind(PreferencesService.class).toInstance(preferencesService);
```

Comment:
I kind of understand why the others wouldn't be needed, but why can this be removed? Is the preferences service bound somewhere else? Do we have any unit tests that make sure preferences are readable in preview? Maybe PreviewDataPipelineTest?

Reply:
Added a test. Discussed offline, TLDR the runners I don't believe should need to be able to access the preferences service; that should be done by the preview manager. Thanks!